### PR TITLE
[Polymer] Vulcanize polymer src html when building sample app

### PR DESCRIPTION
**Pull request by leonhsl**
BUG=XWALK-6868

### CODE REVIEW

**Comment by leonhsl** [comment]
@halton @jondong @DonnaWuDongxia PTAL, Thanks~

Verified OK locally.

File: sample/package_sample.py [hunk 1]
```diff
@@ -61,6 +62,34 @@ def npmInstallExtensions(appRoot, distPath):
...
+def doVulcanize(appRoot):
+  # Install vulcanize if not available
+  if isProgAvailable(VULCANIZE_CMD) is None:
+    cmd = NPM_CMD + ' install -g ' + VULCANIZE_CMD
+    subprocess.check_call(cmd, shell=True)
+
+  cmd = VULCANIZE_CMD + ' ' \
+       + os.path.join("elements", "elements.html") \
+       + ' --inline-scripts --inline-css --out-html elements.vulcanized.html'
+  print "cmd: " + cmd
```
Comment (jondong) [inline sample/package_sample.py:74]: is it for debug use?

Comment (leonhsl) [inline sample/package_sample.py:74]: Yeah and I think it's also useful message to users, just like other places in this file calling `print "cmd: " + cmd`

**Comment by jondong** [comment]
lgtm, merging

**Pull request merged**